### PR TITLE
Integrated four services with proxy server

### PR DESCRIPTION
**Pull request by chelseaSchmidt**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```
Comment (JimKnape) [inline .babelrc:2]: Is there a a need for babel here still?

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <div>Hi</div>
+    <div id="app">Test</div>
+    <div id="image-main"></div>
+    <div id="experiences"></div>
+    <div id="best-nearby"></div>
+    <div id="reviews"></div>
+    <script src="imageMain/bundle.js"></script>
+    <script src="exp/bundle.js"></script>
+    <script src="reviewsModule/bundle.js"></script>
+    <script src="bestNearby/bundle.js"></script>
```
Comment (JimKnape) [inline client/public/index.html:17]: Does the order of this matter for styling at all? It may be nice for the scroll of the page to match the order of the scripts.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "tripadvisor_proxy-server_chelsea",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server/index.js",
+    "watch": "webpack --watch"
```
Comment (JimKnape) [inline package.json:9]: Is this necessary still?

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+const bestNearby = 'http://127.0.0.1:3003';
+const reviews = 'http://127.0.0.1:3004';
+const proxyPort = 3006;
+
+const publicDir = path.resolve(__dirname, '..', 'client', 'public');
+
+app.use(morgan('dev'));
+app.use('/:productId', express.static(publicDir));
+
+// ==============================IMAGE MAIN=============================
```
Comment (JimKnape) [inline server/index.js:20]: So great Chelsea! So clear. 

The only comment i can make is if we can standardize the end points for each component.

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+const path = require('path');
+const srcDir = path.resolve(__dirname, 'client', 'src', 'test.jsx');
```
Comment (JimKnape) [inline webpack.config.js:2]: Is this file necessary?

File: .babelrc (hunk 1 above)
Comment (chelseaSchmidt) [reply .babelrc:2]: Nope good call! All the webpack/babel stuff you caught is old, I will push a commit removing these and then merge

File: client/public/index.html (hunk 2 above)
Comment (chelseaSchmidt) [reply client/public/index.html:17]: Good catch! I will switch the order, that was what I was going for but accidentally mixed them up at some point

File: server/index.js (hunk 4 above)
Comment (chelseaSchmidt) [reply server/index.js:20]: Agreed! After everyone's got something working, we should touch base as a group and agree on what to call everything. I had to make a temporary change in my copy of everyone's repo to get endpoints to match up to get this working
